govcd/nsxv_ipset.go

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright 2019 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"encoding/xml"
+	"fmt"
+	"net/http"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"github.com/vmware/go-vcloud-director/v2/util"
+)
+
+func (vdc *Vdc) CreateNsxvIpSet(ipSetConfig *types.EdgeIpSet) (*types.EdgeIpSet, error) {
```

Comment:
Would like an introduction-like comment about IP sets and into which object they are being created.

Reply:
Indeed. I forgot to add a doc at all. Fixed.